product_docs/docs/biganimal/release/getting_started/creating_a_cluster/index.mdx

```diff
@@ -233,29 +233,27 @@ Enable **Superuser Access** to grant superuser privileges to the edb_admin role.
 Enable **Transparent Data Encryption (TDE)** to use your own encryption key. This option is available for EDB Postgres Advanced Server and EDB Postgres Extended Server for version 15 and later. Select an encryption key from your project and region to encrypt the cluster with TDE. To learn more about TDE support, see [Transparent Data Encryption](../../overview/03_security/#your-own-encryption-key---transparent-data-encryption-tde).
 
 !!!Note "Important"
-- To enable and use TDE for a cluster, the encryption key must be enabled and added at the project level before creating a cluster. 
+- To enable and use TDE for a cluster, you must first enable the encryption key and add it at the project level before creating a cluster. 
   To add a key, see [Adding a TDE key at project level](../../administering_cluster/projects.mdx/#adding-a-tde-key).
 - To enable and use TDE for a cluster, you must complete the configuration on the platform of your key management provider after creating a cluster. See [Completing the TDE configuration](#completing-the-TDE-configuration) for more information.
 !!!
 
 #### Completing the TDE configuration
 
-After you create the cluster in the BigAnimal console, the UI will display the **Waiting for access to encryption key** state. To complete the configuration and enable the key sync between BigAnimal and the key management platform you must grant encrypt and decrypt permissions to your key:
+After you create the cluster in the BigAnimal console, the cluster is in the **Waiting for access to encryption key** state. To complete the configuration and enable the key sync between BigAnimal and the key management platform, you must grant encrypt and decrypt permissions to your key:
```

Comment:
Is this edit okay? I'd rather we stay away from using the UI as the subject.

Reply:
Sounds good! I think with the rebranding into EDB Postgres AI, we could start calling it Console or EDB Postgres AI Console whenever necessary. Here, I like the edit with "cluster".